examples/zoom-provider/babel.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = (api) => {
```

Comment:
I was having some issues around getting the client examples working, so this is still here from that, but I just ended up doing a build and incorporate the zoom package directly into another basic project that way. Let me know if you think it woul dbe helpful to move that client application into this example folder